modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```
@@ -138,6 +139,11 @@ request-string=Request String
 response-string=Response String
 scope-parameter-name=Scope Parameter Name
 search-...=Search...
+search-bar-display-context=Search Bar Display Context
+search-bar-display-context-help=Access the data contained in the SearchResultsPortletDisplayContext object (for example, keywords).
```

Let's make the example a bit more specific like `search-results-display-context-help`.

```
search-results-display-context-help=Access the data contained in the SearchResultsPortletDisplayContext object (for example, its getKeywords returns the current search query).
```

```suggestion
search-bar-display-context-help=Access the data contained in the SearchResultsPortletDisplayContext object (for example, its getKeywords returns the current search query).
```